### ISSUE TITLE
Move Source XandY values always set a value

### ISSUE THREAD

**Issue by lcalder** [nhielost/obs-midi-mg#30]
I have a source which I would like to reposition using two independent cc sliders, one to control the X axis and the other to control the Y axis.
The problem is that there is no way to "deselect" or "ignore" the Pos X or Pos Y so that its value does not overwrite the sources current value, to allow me to change ONLY the X or ONLY the Y using 0-127.
So, for example, when I set Pos X to fixed 0, and Pos Y to --- "0-127" it will indeed adjust my Y axis, but only after resetting the X position to 0. When adjust the other slider (set with Pos X to --- "0-127" and Pos Y to 0) it moves the source along the X axis, but only after resetting my Y value to 0.
It would be great if there were an addition setting in the middle button [Fixed, 0-127, No-change]

![image](https://user-images.githubusercontent.com/50305507/198935021-ae045db8-585f-44e0-ab26-f44c2e4a24e4.png)

**Comment by nhielost** [comment]
Thanks for the feedback!

I agree that this should be implemented. I don't know why I never thought of this!

**Comment by nhielost** [comment]
This issue has been resolved as of the latest update (v2.1.0). If this issue persists, please create another issue.

Again, thanks for the feedback!
